app/config/colors.js

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
I read `config` folder as storage of configuration, .env files etc...
Do you think will be good idea to move these files to `Styles` folder as I recommended above?

Reply:
Yeah, I think Styles would probably be more clear than Config for colors.js. Will refactor that.

Reply:
Done

Reply:
Ty!